diff --git a/lib/index.js b/lib/index.js
index 3b1782af5d5ffb5bf39fa294d736fa22f648dfef..e84bd53a0f93e97d1a0bcd58a9af03b11fce255f 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -248,6 +248,9 @@ export default function({ emit = true } = {}) {
                     sourcesContent,
                 });
 
+                transformResult.map.sources = [];
+                transformResult.map.sourcesContent = [];
+
                 return {
                     ...transformResult,
                     warnings,